lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -532,7 +532,7 @@ lldb::VariableSP SwiftExpressionParser::FindSelfVariable(Block *block) {
   return variable_list_sp->FindVariable(ConstString("self"));
 }
 
-static void AddRequiredAliases(Block *block, lldb::StackFrameSP &stack_frame_sp,
+static bool AddRequiredAliases(Block *block, lldb::StackFrameSP &stack_frame_sp,
```

Comment:
Can you document what the return value means?

Reply:
Should you instead return a Status that contains an error message about what failed?

Reply:
I added the error messages, but I wonder if having them is better than a generic error. They aren't really actionable from the user's point of view.

Reply:
@PortalPete what do you think?

Reply:
I generally think more information is helpful to the community. The error messages certainly help me understand what the code is doing a bit better.

My answer hinges on what you mean by "actionable" and who could take action. Even if the message isn't about the developer's code that they're debugging, it could potentially inform them it's not responsible for the issue at hand.